src/actions/config.js

```diff
@@ -54,47 +53,21 @@ export function asyncConfigInit() {
 export function asyncConfigFetch() {
   return dispatch => {
     dispatch(configFetch());
-    let opts = {};
-    opts.headers = { Accept: 'application/javascript' };
-    let configUrl = './config.js';
-    http.get(configUrl, opts, function(error, response) {
-      if (response) {
-        dispatch(configFetchSuccess());
-        try {
-          let config = JSON.parse(response.text);
-          Object.keys(config).map(function(key) {
-            dispatch(configUpdateByKey(key, config[key]));
-          });
-        } catch (e) {
-          dispatch(notificationAddError(e.message + ' ' + configUrl));
-        }
-        dispatch(asyncUserConfigFetch());
-      } else {
-        dispatch(configFetchError());
-      }
-    });
-  };
-}
-
-export function asyncUserConfigFetch() {
-  return dispatch => {
-    dispatch(configFetch());
-    let opts = {};
-    opts.headers = { Accept: 'application/javascript' };
-    let configUrl = './userConfig.js';
-    http.get(configUrl, opts, function(error, response) {
+    configGet(function(error, response) {
```

Comment:
I think we should move this into `actions/pluginSettings.js` with the rest of the async action creators for the plugins API.  This might mean we can delete `actions/config.js`.